pkg/cmd/roachtest/tests/pg_regress.go

```diff
@@ -472,6 +497,8 @@ func registerPGRegress(r registry.Registry) {
 		CompatibleClouds:  registry.AllExceptAWS,
 		Suites:            registry.Suites(registry.Weekly),
 		Leases:            registry.MetamorphicLeases,
+		// TODO(#142777): matview file results in an invalid descriptor.
+		SkipPostValidations: registry.PostValidationInvalidDescriptors,
```

Comment:
the bug has been fixed: https://github.com/cockroachdb/cockroach/pull/143287

Reply:
Thanks!